README.md

```diff
@@ -116,6 +116,12 @@ framework:
     validation: { enable_annotations: true }
 ```
 
+### Testing
+
+When installed as a standalone library, inside the vendor folder run:
+
+`php bin/phpunit chesscom/diagram-generator/`
```

Comment:
The second part of this command is not needed, as we have defined a `phpunit.xml.dist` on the root indicating where the tests are.

Also, the correct command to run would be: `php vendor/bin/phpunit`